master/internal/experiment.go

```diff
@@ -765,6 +765,26 @@ func (e *experiment) processOperations(
 				e.syslog.Errorf("trial %s already exists", op.RequestID)
 				continue
 			}
+			var continueFromTrialID *int
+			if e.continueTrials {
+				trial, err := db.TrialByExperimentAndRequestID(context.TODO(), e.ID, op.RequestID)
```

Comment:
nit https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

can we have the errors.Is(err, sql.ErrNoRows) / err != nil case separate from the err == nil in the style of ^

Reply:
Done